Marlin/ubl_motion.cpp

```diff
@@ -589,6 +589,10 @@
 
       // Otherwise perform per-segment leveling
 
+      #if ENABLED(ENABLE_LEVELING_FADE_HEIGHT)
+        const float fade_scaling_factor = ubl.fade_scaling_factor_for_z(ltarget[Z_AXIS]);
+      #endif
```

Comment:
Yes, and that is how I noticed the other half of the fade semantics had disappeared during one of my many git merge/rebase sessions. So thanks for removing it!  That's how I noticed something was wrong.